app/models/calculated_product_test.rb

```diff
@@ -54,14 +54,15 @@ def generate_records
   end
 
   def validators(doc)
-    @validators ||= [::Validators::QrdaCat3Validator.new(expected_results),
-      ::Validators::MeasurePeriodValidator.new(),
-      ::Validators::ExpectedResultsValidator.new(expected_results)]
+    @validators ||= [
+      ::Validators::QrdaCat3Validator.new(expected_results),
+      ::Validators::MeasurePeriodValidator.new,
+      ::Validators::ExpectedResultsValidator.new(expected_results)
```

Comment:
Doesn't look like there's any changes here outside of Rubocop issues? If so, can you revert to keep the branching from upstream small?